examples/terraform/openstack/control_plane.tf

```diff
@@ -37,6 +42,10 @@ resource "openstack_compute_instance_v2" "control_plane" {
   security_groups = [openstack_networking_secgroup_v2.securitygroup.name]
   config_drive    = var.config_drive
 
+  scheduler_hints {
+    group = openstack_compute_servergroup_v2.control_plane.id
+  }
```

Comment:
Can we provide an easy way to disable this? For example, if we can control this with a variable.